datalake/src/main/java/com/sequenceiq/datalake/service/SdxEventsService.java

```diff
@@ -131,7 +131,7 @@ private String getAccountId(String crnString) {
     private SdxCluster getDatalake(String environmentCrn) {
         LOGGER.info("Looking for datalake associated with environment Crn {}", environmentCrn);
         List<SdxCluster> sdxClusters = sdxService.listSdxByEnvCrn(environmentCrn);
-        sdxClusters.stream().forEach(sdxCluster -> LOGGER.info("Found SDX cluster {}", sdxCluster));
+        sdxClusters.forEach(sdxCluster -> LOGGER.info("Found SDX cluster {}", sdxCluster));
```

Comment:
Should this be logged in a single message rather than a separate log entry for cluster?

Reply:
The idea is log one object per log message.

Reply:
I think it might have the potential to be noisy, but it seems to be a rare case